src/ewatercycle/base/model.py

```diff
@@ -83,6 +83,10 @@ def _check_parameter_set(self):
     def _make_bmi_instance(self) -> OptionalDestBmi:
         """Attach a BMI instance to self._bmi."""
 
+    # this has different signature than pymt see
+    # https://github.com/csdms/pymt/blob/9ef61a0010b4997d5a2b09e5d434371598291261/pymt/framework/bmi_setup.py#L77C21-L77C32
+    # where it is {}.items()
+    # TODO is this OK?
```

Comment:
Might be better to conform to pymt, can you do that in the PR with the abstract model tests?

Reply:
Done in 8c44672e6319bf60328cea08ee5c75400bb45aeb